src/main/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/model/translator.kt

```diff
@@ -67,10 +67,10 @@ fun Prisoner.translate(existingPrisoner: Prisoner?, ob: OffenderBooking, incenti
   ob.physicalMarks?.forEach { pm ->
     pm.bodyPart?.let { BodyPartDetail(it, pm.comment) }?.let { bodyPart ->
       when (pm.type) {
-        "TAT" -> this.tattoos = this.tattoos?.plus(bodyPart) ?: listOf(bodyPart)
```

Comment:
turns out that Prison API returns the description rather than the code

Reply:
Better to fix prison API?